ui/app/src/components/SingleFileUpload/SingleFileUpload.tsx

```diff
@@ -342,14 +355,35 @@ export function SingleFileUpload(props: SingleFileUploadProps) {
       </form>
       <div className="uppy-progress-bar" />
       <div className="uploaded-files">
-        <Typography variant="subtitle1" component="h2" className={classes.description}>
```

Comment:
Looks like classes.description is no longer in use. Delete.

Reply:
yeah, already removed https://github.com/craftercms/studio-ui/pull/3991/files#diff-97d6b37d1903dc757211f4bd460923f9a31f8bae376739fde8027950f764efa6L76